application/backend/app/models/label.py

```diff
@@ -14,3 +14,12 @@ class LabelReference(BaseModel):
             }
         }
     }
+
+
+class Label(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+    id: UUID
+    project_id: UUID
```

Comment:
Is there any reason to include `project_id` in the model? I don't think we ever need to resolve the project from a label, and it makes the initialization slightly more complex (you had to update several test files to add this new field).